pkg/template/servicebroker/test-scripts/bind.sh

```diff
@@ -16,6 +17,7 @@ curl \
   -X PUT \
   -H 'X-Broker-API-Version: 2.9' \
```

Comment:
Looks like it is currently targeted for 2.12; hopefully this link renders to the right line in spec.md of the servicebroker pull:  https://github.com/openservicebrokerapi/servicebroker/pull/222/files#diff-958e7270f96f5407d7d980f500805b1bR105

good catch / will switch